PowerUp/app/src/main/res/layout/intro_screen_2.xml

```diff
@@ -0,0 +1,34 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background"
+    tools:context="powerup.systers.com/IntroActivity">
+
+    <ImageView
```

Comment:
You should create a relativeLayout here so you can position the elements in respect to one another instead of using margins for all